src/parsers/aws.ts

```diff
@@ -122,36 +129,64 @@ export const eventBridgeParser = (requestData, responseData) => {
 };
 
 export const sqsParser = (requestData, responseData) => {
-  const { body: reqBody } = requestData;
+  const { body: reqBody } = requestData || {};
```

Comment:
Let's split this function into 2, XML & JSON, to make it more readable

Reply:
Fixed